### DISCUSSION TITLE
Access Denied and Ask about 10.0.3.199 / 10.0.3.198

### DISCUSSION THREAD

**Discussion by HoangGuruu** [ecoricemon/kube#6] [Q&A]
# 1
![image](https://github.com/ecoricemon/auto-kube/assets/111829092/273e2bad-eabd-464b-9389-43b4b5dbdd4b)
- I installed openssh-server in vmdesk
- And have NatNetwork
- Configure ip4v like that : 
![image](https://github.com/ecoricemon/auto-kube/assets/111829092/a3af0b68-6ddb-40a3-a870-7d0bc75568eb)
But access denid : Can you give a solution , please 

# 2 
![image](https://github.com/ecoricemon/auto-kube/assets/111829092/9926fd1e-b9aa-4fcd-b1ea-444958e41aa0)
- I can access with 10.0.3.11:port when i check by :kubectl get all -n argocd
- But can't access with 10.0.3.188 or 10.0.3.199
- Why ? and can you show me a solution

**Comment by ecoricemon** [comment]
Your IP settings are probably not set correctly.
I suggest you to check out your IP settings 

1. MetalLB address pool setting
> kubectl describe -n metallb-system ipaddresspools.metallb.io
- Expect
> Addresses: 10.0.3.100-10.0.3.199

2. Dashboard external IP setting
> kubectl describe -n kubernetes-dashboard services/kubernetes-dashboard | grep LoadBalancer
- Expect
> LoadBalancer ... : 10.0.3.199

3. ArgoCD external IP setting
> kubectl describe -n argocd services/argocd-server | grep LoadBalancer
- expect
> LoadBalancer ... : 10.0.3.198
